fluent-syntax/src/parser.js

```diff
@@ -416,34 +415,37 @@ export default class FluentParser {
   getValue(ps) {
     if (ps.currentIs("{")) {
       ps.peek();
-      ps.peekInlineWS();
-      if (ps.isPeekNextLineVariantStart()) {
+      ps.peekBlank();
+      if (ps.currentPeek() === "*" || ps.currentPeek() === "[") {
```

Comment:
So, this is a bit more tricky. The situation is unusual in this place because we want to try to get a variant list, but if that doesn't work, we bail *back to the `{`* and go for `getPattern`.

That means that if we transition to use `isPeekNextLineVariantStart` I need to complicate it to not always skip.